pkg/apis/node/types.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package node
 
 import (
+	v1 "k8s.io/kubernetes/pkg/apis/core"
```

Comment:
don't call this v1, since it's the internal type. You could leave it as `core` or `coreapi`